python/lsst/metaserv/metaREST_v0.py

```diff
@@ -34,17 +34,28 @@
 
 from flask import Blueprint, request
 import json
+import logging as log
 
-from lsst.db.db import Db
+from lsst.db.dbPool import DbPool
 
 metaREST = Blueprint('metaREST', __name__, template_folder='metaserv')
 
-# connect to the metaserv database
-db = Db(read_default_file="~/.lsst/dbAuth-metaServ.txt")
+# Connect to the metaserv database. Note that the metaserv typically runs for
+# a long time, and the connection can timeout if there long period of inactivity.
+# Use the DbPool, which will keep the connection alive.
+dbPool = DbPool()
+dbPool.addConn("c1", read_default_file="~/.lsst/dbAuth-metaServ.txt")
+
 
 def runDbQuery1(query, optParams=None, notFoundMsg='Not found'):
-    '''Runs query that returns one row.'''
-    row = db.execCommand1(query, optParams)
+    '''Runs query that returns one row. It can raise DbException or mysql
+    exception.'''
+    cursor = dbPool.getConn("c1").getCursor()
+    log.debug("Executing '%s', optParams: %s.", query,
+              ', '.join(optParams) if optParams else 'None')
```

Comment:
This has the same problem that we already encountered elsewhere - if `optParams` contains non-string objects this will result in exception. It's better to pass `optParams` as is and let formatting to take care of the representation.